napari/resources/stylesheet.qss

```diff
@@ -1000,22 +1000,147 @@ QScrollBar::sub-page {
 
 /* ------------- DimsSliders --------- */
 
-QtDims > QRangeSlider[last_used=true] {
+QtDimSliderWidget > QRangeSlider[last_used=true] {
   qproperty-barColor: {{ secondary }};
   qproperty-handleColor: {{ secondary }};
   qproperty-handleBorderColor: {{ secondary }};
 }
 
-QtDims > QRangeSlider[last_used=false] {
+QtDimSliderWidget > QRangeSlider[last_used=false] {
   qproperty-barColor: {{ highlight }};
   qproperty-handleColor: {{ highlight }};
   qproperty-handleBorderColor: {{ highlight }};
 }
 
-QtDims > QScrollBar::handle[last_used=false]:horizontal {
+QtDimSliderWidget > QScrollBar::handle[last_used=false]:horizontal {
     background: {{ highlight }};
 }
 
-QtDims > QScrollBar::handle[last_used=true]:horizontal {
+QtDimSliderWidget > QScrollBar::handle[last_used=true]:horizontal {
     background: {{ secondary }};
 }
+
+QtDimSliderWidget > QScrollBar:left-arrow:horizontal {
+    image: url(":/icons/{{ folder }}/step_left.svg");
+}
+
+QtDimSliderWidget > QScrollBar::right-arrow:horizontal {
+    image: url(":/icons/{{ folder }}/step_right.svg");
+}
+
+
+QtPlayButton {
+    /* border: none; */
+    border-radius: 3px;
+    min-height: 10px;
+    max-height: 10px;
+    min-width: 10px;
+    max-width: 10px;
+    margin: 0px 2px;
+    padding: 2px;
+}
+
+QtPlayButton[reverse=True] {
+    image: url(":/icons/{{ folder }}/left_arrow.svg");
+}
+
+QtPlayButton[reverse=False] {
+    image: url(":/icons/{{ folder }}/right_arrow.svg");
+}
+
+QtPlayButton[playing=True] {
+    image: url(":/icons/{{ folder }}/new_image.svg");
+    background-color: rgb(153, 18, 31);
```

Comment:
do we want to make this color part of our palette, or do we want to hard code it? i'd probably just make it part of the palette, it is pretty easy to do and then no colors are hard coded, but maybe it's a special case

Reply:
sure.  I'll add a "warning" color to the palette.